core-bundle/src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -418,7 +418,7 @@ protected function createExtract()
 
 					if (isset($config['foreignKey']))
 					{
-						$table = substr($config['foreignKey'], 0, strrpos($config['foreignKey'], '.'));
+						$table = substr($config['foreignKey'], 0, strpos($config['foreignKey'], '.'));
```

Comment:
Maybe use a shorter version here?
```suggestion
						$table = explode('.', $config['foreignKey'])[0];
```